core/domain/prod_validators.py

```diff
@@ -105,6 +109,7 @@
 VALID_SCORE_CATEGORIES_FOR_TYPE_QUESTION = [
     '%s\\.[A-Za-z0-9-_]{1,%s}' % (
         suggestion_models.SCORE_TYPE_QUESTION, base_models.ID_LENGTH)]
+ALLOWED_SENDER_EMAIL_ADDRESSES = ['noreply@oppia.org', 'admin@oppia.org']
```

Comment:
In the list, refer to the constants in feconf. Don't hardcode oppia.org stuff all over the code.

Reply:
Done.

Reply:
No such constants exist in feconf or elsewhere. I think there are added while releases. So, it is better to have them just in the file.

Reply:
Ah, good point.

In that case, let's drop this validation check. I still think we shouldn't have deployment-specific constants all over the codebase, and since the main thing we should be checking is model integrity, I don't think we need this check.

Reply:
Done.